File: projects/data structures and algorithms in c/linklists/problem_one.cpp
#include<iostream>
using namespace std;

void twoSum(int nums[],int target){
  
    int first =0;
for ( int i = 1; i < 3; i++)
{
   if(nums[first] +nums[i] == target){
    cout<<"the indices are: "<<first<<","<<i<<endl;

   }
   else{
    first=i;
   }
}
    }
    

int main(){

    int numbers[3]={3,2,4};
    // [3,2,4]
    int target =6;
    twoSum(numbers,target);


    return 0;
}

File: projects/data structures and algorithms in c/linklists/problem_one.py
def twoSum( nums, target):
      
        nums_len =len(nums)
   
       
        for i in range(nums_len):
            for j in range(i+1, nums_len):
            
               if(nums[j]+nums[i]==target):
                  print([i,j])
                 
                 
                  
              
       

       #my mistake was looping the list and not skipping the first element bescause i already selected it

        


numbers=[3,2,3]
target=6
twoSum(numbers,target)